index.html

```diff
@@ -0,0 +1,1249 @@
+                        ;" class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!-- Ratings -->
+
+            <!-- Grids -->
+
+            <section id="grid">
+                <div class="code-container">
+                    <h2 class="component-title">Grids-2</h2>
+                    <div class="grid-2">
+                        <div class="col">1</div>
+                        <div class="col">2</div>
+                    </div>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%2874%2C118%2C159%2C1%29&t=seti&wt=none&l=auto&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-2%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522col%2522%253E1%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522col%2522%253E2%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                        " class="iframe-container" sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+
+                <div class="code-container">
+                    <h2>Grids-3</h2>
+                    <div class="grid-3">

```

Comment:
Suggestion - grid section borders are colliding together which makes intermediate line solid try to remove that.

Reply:
done